src/pages/settings/Wallet/PaymentMethodList.tsx

```diff
@@ -279,7 +282,10 @@ function PaymentMethodList({
                     key: card.cardID.toString(),
                     // eslint-disable-next-line @typescript-eslint/prefer-nullish-coalescing
                     title: card?.nameValuePairs?.cardTitle || card.bank,
-                    description: getDescriptionForPolicyDomainCard(card.domainName),
+                    description:
+                        !card?.nameValuePairs?.issuedBy || !card?.lastFourPAN
+                            ? getDescriptionForPolicyDomainCard(card.domainName)
+                            : `${card?.lastFourPAN} ${CONST.DOT_SEPARATOR} ${getDescriptionForPolicyDomainCard(card.domainName)}`,
```

Comment:
I can still see it 
<img width="710" alt="image" src="https://github.com/user-attachments/assets/143b117d-edcb-479b-a7bc-628c2c31c0ad" />


the card even though  it is a physical has the `issuedBy` property as its admin issued card - we need to update the else part to only show the card number and dot separator if applicable

Reply:
Good catch!